src/common/transformations/include/ov_ops/type_relaxed.hpp

```diff
@@ -300,8 +300,7 @@ std::shared_ptr<Node> TypeRelaxed<BaseOp>::clone_with_new_inputs(const OutputVec
         auto origin_input_type = get_origin_input_type(i);
         if (origin_input_type == element::undefined)
             origin_input_type = BaseOp::get_input_element_type(i);
-        fake_new_inputs.push_back(
```

Comment:
As I see , we reconnect all inputs to the new args and only after that do "validate_and_infer_type" here
<img width="200" alt="image" src="https://github.com/user-attachments/assets/6c5abd23-4ba8-4b28-9d4a-bf5a6e7b2761">
So we use the new shapes in the validate function.
Not so sure I understand the case, could you add a test please? Could you also provide a error log?

Reply:
I met the error while executing Stable Diffusion 3 int8 quantized static model on GPU.

![image](https://github.com/user-attachments/assets/bb0ecd48-6c97-4dbc-83d4-ece3029fe91a)
The `MoveEltwiseUpThroughDataMovPerChannel` transformation pass tries to update the left subgraph into the right one.

https://github.com/openvinotoolkit/openvino/blob/17fa918f1868c4c80e8395149d4b8a98ff8d0f5e/src/common/transformations/src/transformations/common_optimizations/move_eltwise_up_data_movement.cpp#L209-L216

At L211, "mul_const_0" is replaced by "mul_const_1". So, at this point the shapes of the 1st and 2nd inputs of "multiply_0" are [1, 3, 4, 5] and [1, 3, 1].
At L215, "eltwise_inputs[0]" (1st input of "multiply_0") is replaced by input (f32:[1, 3, 20]).
Please note that the shape of 1st input of "multiply_0" is still [1, 3, 4, 5].

https://github.com/openvinotoolkit/openvino/blob/17fa918f1868c4c80e8395149d4b8a98ff8d0f5e/src/common/transformations/include/ov_ops/type_relaxed.hpp#L303-L304

So, `BaseOp::get_input_partial_shape(0)` is [1, 3, 4, 5] and `new_args[0]` is [1, 3, 20].
This makes the below error.

> `[ ERROR ] Exception from src/inference/src/cpp/core.cpp:104:
Exception from src/inference/src/dev/plugin.cpp:53:
Check 'TRShape::broadcast_merge_into(output_shape, input_shapes[1], autob)' failed at src/core/shape_inference/include/eltwise_shape_inference.hpp:26:
While validating node 'opset1::Multiply Multiply_330895 (opset1::Parameter Parameter_330893[0]:f32[1,1178,24,64], opset1::Parameter Parameter_330894[0]:f32[1,1178,1]) -> (dynamic[...])' with friendly_name 'Multiply_330895':
Argument shapes are inconsistent.
`

I also added a unit test for this case.

Reply:
ok, it looks reasonable.
 Approved.